docs-mk/fr/docs/guide/messenger.md

```diff
@@ -37,26 +43,67 @@ _Paramètres > Général > Visibilité de la page > **Non publiée**_
 
 * _Ajouter une app_
 
+![Créer une application partie 1](https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/create-app-0.png)
+
+* _Créer une app_ > *Gérer les intégrations professionnelles*
+
+![Créer une application partie 2](https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/create-app-1.png)
+
 * Entrez un nom pour l'_application_
 
-* _Créer un ID d'app_... Notez l'ID de la page, vous en aurez besoin plus tard.
+<img src="https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/create-app-2.png" alt="Créer une application partie 3" style="zoom:50%;" />
 
 * Ajoutez un produit : _Messenger_
 
-* Dans les paramètres, générez un jeton (_token_) pour votre page. Notez ce _token_ pour la suite.
+![Ajouter messenger à une application partie 1](https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/add-messenger-page-0.png)
+
+<img src="https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/add-messenger-page-1.png" alt="Ajouter messenger à une application partie 2" style="zoom: 50%;" />
+
+## Créer un connecteur Messenger
+
+* Dans *Tock Studio* allez dans *Settings* > *Configurations* :
+
+<img src="https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/create-connector-0.png" alt="Créer un connecteur messenger partie 1" style="zoom:50%;" />
+
+* Créez un connecteur de type *Messenger* et ouvrez la section *Connector Custom Configuration*
+
+<img src="https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/create-connector-1.png" alt="Créer un connecteur messenger partie 2" style="zoom:50%;" />
+
+### Remplir les champs
+
+<img src="https://raw.githubusercontent.com/theopenconversationkit/theopenconversationkit.github.io/master/fr/images/doc/connector-messenger/connect-tock-0.png" alt="Connecter tock partie 1" style="zoom:50%;" />
+
+### 1️⃣ Id d'application
+
+Allez sur https://developers.facebook.com, sur la page de votre application :
```

Comment:
URL should be a real link: `[https://developers.facebook.com](https://developers.facebook.com)`